web/src/leagues.json

```diff
@@ -1,90 +1,112 @@
 [
   {
-    "id": 22,
-    "name": "Winter Indoor 2024/2025 Winter Sunday Parity League"
+    "id": "9",
```

Comment:
I made these strings because a lot of the typing was already this way (I think because it came from url params). This breaks the sort on the league picker in the StatKeeper but that one will go away. It might have affected the real app too.

Reply:
Yes it did.